src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/model/ChangePolicy.kt

```diff
@@ -26,7 +26,8 @@ import java.io.IOException
 
 data class ChangePolicy(
     val policyID: String,
-    val state: String?
+    val state: String?,
+    val include: List<StateFilter>
 ) : ToXContentObject {
 
     override fun toXContent(builder: XContentBuilder, params: ToXContent.Params): XContentBuilder {
```

Comment:
Need to add `include` field into the `XContent`.

Reply:
Did not add include as I do not want to save it on the document, this is just used in the REST API for filtering the indices. Can add a comment.